inspire/modules/workflows/tasks/submission.py

```diff
@@ -91,6 +91,7 @@ def _create_ticket(obj, eng):
             email=email,
             title=d.title,
             identifier=sip.metadata.get("system_number_external", {}).get("value", ""),
+            user_comment=sip.metadata.get("hidden_note", {}).get("value", ""),
```

Comment:
Then we should make it a list on inspire.cfg, right?
```
url:
    schema:
        {'url': {'type': 'list', 'force': True}}
...
```

Anyway something like this should work
`filter(lambda k: 'submitter' == k['source'], sip.metadata.get("hidden_note", []))`